packages/jsii-pacmak/bin/jsii-pacmak.ts

```diff
@@ -54,6 +58,11 @@ import { VERSION_DESC } from '../lib/version';
       desc: 'force generation into a target-named subdirectory, even in single-target mode',
       default: true,
     })
+    .option('force-target', {
+      type: 'boolean',
+      desc: 'force generation of the given targets, even if the source package.json doesnt declare it',
```

Comment:
Hah. Well the Ruby runtime depended on it (wanted to generate ruby code for jsii-calc and friends, even though it wasn't declared and is otherwise untested). My first instinct was to get rid of the ruby runtime, but since I already overstepped my bounds a bit by removing the Sphinx target (ooohhh) I didn't want to make the scope of this PR even bigger and risk incurring more discussion.

I also did not want to enshrine the current Ruby generation into jsii-pacmak's `expected` tests, so this seemed like the smallest delta to keep the repository building.

Reply:
Oh I see. I wouldn't have opposed dropping Ruby (having side concerns about he lost effort, but that's another discussion).

Regardless - maybe just mention in the description that `--force-target` might cause `jsii-pacmak` to exit in error.